rc/base/ctags.kak

```diff
@@ -7,9 +7,12 @@ decl str-list ctagsfiles 'tags'
 
 def -params ..1 \
     -shell-candidates '
+        realpath() {
+            printf %s\\n "$(cd -P -- "$(dirname -- "$1")" && printf %s\\n "$PWD/${1##*/}")"
```

Comment:
One to print the expanded `$PWD` in the command expansion scope and the other to print the result of `realpath`.
If you have a better way to do it I'll take it, especially the shell expansion in the shell expansion.